GOVERNANCE.md

```diff
@@ -120,10 +123,29 @@ the issue tracker is:
 
 ## Collaborator nominations
 
-Existing Collaborators can nominate someone to become a Collaborator. Nominees
-should have significant and valuable contributions across the Node.js
+### Who can nominate Collaborators ?
+
+Existing Collaborators can nominate someone to become a Collaborator.
+
+### Ideal Nominees
+
+Nominees should have significant and valuable contributions across the Node.js
 organization.
 
+Contributions can be:
+
+* Commits
+* Pull requests
+* Comments
+* Reviews
+* Helping others
+* Issues
+* Participation in other projects, teams, and working groups of the Node.js
+organization
+* Other participation in the wider Node.js community
+
+### Nominating a new Collaborator
+
```

Comment:
I splitted this part to make it friendly for both collaborators and non-collaborators .